app/js/lib/foxbox/services/motion-sensor.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+import BaseService from './base';
+
+const TYPE = 'motion-sensor';
+
+export default class MotionSensorService extends BaseService {
+  constructor(props, config) {
+    super(props, config);
+    Object.seal(this);
+  }
+
+  get type() {
+    return TYPE;
+  }
```

Comment:
Well.... I intentionally didn't add it, it's somewhat special thing, that will automatically change in 30 seconds (configurable though). My rationale was - until we get reliable method of keeping this properties in sync with foxbox there is a little value in them, they will be just confusing...

But if you mean method not getter/setter property, I can add but still won't use it until we get polling/websocket support - otherwise people will complain for half-working feature :)